frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx

```diff
@@ -1,73 +1,171 @@
+        name: 'Alice Johnson',
+        image: 'https://placekitten.com/52/52',
+    },
+    {
+        value: 'Bob Brown',
+        name: 'Bob Brown',
+        image: 'https://placekitten.com/53/53',
+    },
+    {
+        value: 'Charlie Green',
+        name: 'Charlie Green',
+        image: 'https://placekitten.com/54/54',
+    },
+];
+
+const categories = [
+    // props to Doughnut Chart
+    {
+        value: 'shopping',
+        name: 'Shopping',
+        image: 'https://img.freepik.com/free-photo/girl-holds-fashion-shopping-bag-beauty_1150-13673.jpg',
+    },
+    {
+        value: 'food',
+        name: 'Food',
+        image: 'https://media.istockphoto.com/id/153010865/uk/%D1%84%D0%BE%D1%82%D0%BE/%D1%87%D0%B5%D1%80%D0%B2%D0%BE%D0%BD%D0%B5-%D1%8F%D0%B1%D0%BB%D1%83%D0%BA%D0%BE.jpg?s=612x612&w=0&k=20&c=Hr4iJ9Taaiaz4SLEAX7BbFSvTsnSrgg-KlRV4IIHG4s=',
+    },
+    {
+        value: 'games',
+        name: 'Games',
+        image: 'https://img.freepik.com/free-vector/joystick-game-sport-technology_138676-2045.jpg?w=2000',
+    },
+    {
+        value: 'car',
+        name: 'Car',
+        image: 'https://store-images.microsoft.com/image/apps.27729.9007199266312409.d86b04ff-7ebb-4451-a920-9cfe5dbd05d7.ffd1ea39-da23-4062-936a-66afe45ffe14?mode=scale&q=90&h=200&w=200&background=%23000000',
+    },
+    {
+        value: 'travel',
+        name: 'Travel',
+        image: 'https://thumbs.dreamstime.com/b/travel-world-landmarks-background-blue-sky-46083021.jpg',
+    },
+];
+
 const WalletDetails: React.FC = () => {
     const { control, errors } = useAppForm<{ note: string }>({
         //It needs to change
         defaultValues: DEFAULT_INPUT,
     });
 
+    const [selectedPeopleMulti, setPeopleSelectedMulti] = useState<
+        MultiValue<DataType> | SingleValue<DataType>
+    >([]);
+
+    const [selectedCategoriesMulti, setCategoriesSelectedMulti] = useState<
+        MultiValue<DataType> | SingleValue<DataType>
+    >([]);
+
+    const rangeLimits = { min: -100, max: 1000 };
+    const [currentRange, setCurrentRange] = useState(rangeLimits);
+    const [, setFilteredData] = useState(mockSliderData);
+
+    const handlePeopleMultiDropdownChange = useCallback(
+        (selectedOption: MultiValue<DataType> | SingleValue<DataType>) => {
+            if (selectedOption === null) {
+                setPeopleSelectedMulti([]);
+            } else {
+                setPeopleSelectedMulti(selectedOption);
+            }
+        },
+        [],
+    );
+
+    const handleCategoriesMultiDropdownChange = useCallback(
+        (selectedOption: MultiValue<DataType> | SingleValue<DataType>) => {
+            if (selectedOption === null) {
+                setCategoriesSelectedMulti([]);
+            } else {
+                setCategoriesSelectedMulti(selectedOption);
+            }
+        },
+        [],
+    );
+
+    const handleSliderChange = useCallback(
+        (range: { min: number; max: number }): void => {
```

Comment:
Check if there is interface for `{ min: number; max: number }`, if not, create it

Reply:
fixed